include/no_os_spi.h

```diff
@@ -114,6 +114,25 @@ struct no_os_spi_msg {
  */
 struct no_os_spi_platform_ops ;
 
+/**
+ * @struct no_os_spibus_init_param
+ * @brief Structure for SPI bus initialization parameter
+*/
+struct no_os_spibus_init_param {
+	/** SPI bus chip ID */
```

Comment:
change the comment here to something like "SPI bus number (0 for SPI0, 1 for SPI1 etc.)"